docs/docs/installation/configuration/namespace-keptn.md

```diff
@@ -0,0 +1,78 @@
+---
+comments: true
+---
+
+# Namespaces and Keptn
+
+This page discusses how to restrict the namespaces
+where Keptn and the
+[lifecycle-operator](../../../components/lifecycle-operator)
+run.
+
+For information about how to allocate Keptn resources
+across namespaces, see
+[Namespaces and resources](namespace-resources.md).
+
+## Default behavior
+
+Keptn must be installed on its own namespace
+that does not run any other components,
+especially any application deployment.
+
+By default, Keptn lifecycle orchestration is enabled
+for all namespaces except the followings:
+
+- `kube-system`
+- `kube-public`
+- `kube-node-lease`
+- `cert-manager`
+- `keptn-system`
+- `observability`
+- `monitoring`
+- `<Keptn installation namespace>`
+
+## Custom namespace restriction
+
+If your cluster has other namespaces
+that the Keptn lifecycle orchestration should ignore,
+you must:
+
+- Annotate the namespaces where Keptn lifecycle orchestration is allowed
+- Modify the Helm chart to specify the namespaces
+  that Keptn lifecycle orchestration is allowed.
```

Comment:
I would rephrase this, something like

If you want to restrict Keptn only to certain namespaces, you should:

- allow those namespaces during installation
- annotate those namespaces

I would maybe vote for re-using the test we already have in the readme